website/docs/gas.md

```diff
@@ -0,0 +1,33 @@
+# Gas fees update
+
+The gas fees update endpoint allows users to modify the current gas prices on a running devnet. This feature is particularly useful for testing purposes and for adjustments needed after forking to align with the forked network's gas prices.
+
+JSON-RPC Request
+The following JSON-RPC request can be used to update gas prices:
+
+```
+JSON-RPC
+{
+    "jsonrpc": "2.0",
+    "id": "1",
+    "method": "devnet_updateGas"
+    "params": {
+        "gas_price_wei": 10000,
+        "data_gas_price_wei": 10000,
+        "gas_price_strk": 10000,
+        "data_gas_price_strk": 10000,
+    }
+}
+```
+
+Response
+The expected response from the server will mirror the request parameters, confirming the updated gas prices:
+
+```
+{
+    "gas_price_wei": 10000,
+    "data_gas_price_wei": 10000,
+    "gas_price_strk": 10000,
+    "data_gas_price_strk": 10000,
```

Comment:
IMO this essential equality of request and response is unnecessary. It would be cool to return all of the gas prices if the request didn't require all of them. E.g. if the user would be allowed to specify just the one they want to change, then the response including all of them would make sense.

Reply:
I'll change that to allow just a change of one gas parameter but you will receive all of them.

Reply:
e3744dff1e7b3b4fed9f507f17818d21776a45a2